src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -322,25 +325,29 @@ private boolean isEligibleForContribution(UUID centralServerId, Instance instanc
   }
 
   private boolean isContributed(UUID centralServerId, Instance instance) {
+    log.info("isContributed:: parameters centralServerId: {}, instance id: {}", centralServerId, instance.getId());
     return recordContributionService.isContributed(centralServerId, instance);
   }
 
   private boolean isEligibleForContribution(UUID centralServerId, Item item) {
+    log.info("isEligibleForContribution:: parameters centralServerId: {}, item id: {}", centralServerId, item.getId());
     return validationService.isEligibleForContribution(centralServerId, item);
   }
 
   private boolean isContributed(UUID centralServerId, Instance instance, Item item) {
+    log.info("isContributed:: parameters centralServerId: {}, instance id: {}, item id: {}", centralServerId, instance.getId(), item.getId());
     return recordContributionService.isContributed(centralServerId, instance, item);
   }
 
   private void contributeInstanceItems(UUID centralServerId, Instance instance, Statistics stats) {
+    log.info("contributeInstanceItems:: parameters centralServerId: {}, instance id: {}, stats: {}", centralServerId, instance.getId(), stats);
```

Comment:
Does Statistics have a toString method ? Will it print the needed detail or just the data type ? Please check otherwise no point in printing this object alone, you might need a toString method to print its internal meaningful information.

Reply:
It does have toString() Method, but do we really need Statistics in this case? If not, can I remove it?